.travis.yml

```diff
@@ -118,6 +118,7 @@ before_install:
 
 install:
   - yarn install
+  - node ./bin/packages-for-commit.js
```

Comment:
How come we don't use yarn for running this script?

Reply:
yarn workspaces has a bug with bin files in the root package not getting put into the path correctly. I hoisted these files to root because they are run on all packages and seemed to be messing with tarball generation when inside a package we were packing (I'm not sure they actually were, but architecturally they made more sense hoisted anyway so I went ahead and pulled them out just in case, ultimately `yarn pack` just wan't working and `npm pack` is)

Reply:
I suspect I can move it to yarn by deleting my local install entirely and doing a fresh checkout to force the bin files to regenerate, as I think the issue I encountered with bit files in root of a workspace is only for updates and not fresh checkouts.

Reply:
Tried that and bin files still seem to be borked so going to leave as is for now